src/Symfony/Bundle/DependencyInjection/Configuration.php

```diff
@@ -534,7 +534,9 @@ private function addOpenApiSection(ArrayNodeDefinition $rootNode): void
                             ->end()
                             ->info('To pass extra configuration to Swagger UI, like docExpansion or filter.')
                         ->end()
-                        ->booleanNode('overrideResponses')->defaultTrue()->info('Whether API Platform adds automatic responses to the OpenAPI documentation.')
+                        ->booleanNode('overrideResponses')->defaultTrue()->info('Whether API Platform adds automatic responses to the OpenAPI documentation.')->end()
+                        ->scalarNode('error_resource_class')->defaultNull()->info('The class used to represent errors in the OpenAPI documentation.')->end()
```

Comment:
shouldn't it be camel cased to match other configuration names?

Reply:
Symfony use snake case for configs. In the current section, we have both `swagger_ui_extra_configuration ` and `overrideResponses`.

It will be nice to use snake case for new options and rename options with camel case in 5.0.